doc/classes/Viewport.xml

```diff
@@ -71,6 +71,12 @@
 			<description>
 			</description>
 		</method>
+		<method name="get_screen_transform" qualifiers="const">
+			<return type="Transform2D" />
+			<description>
+				Returns the transform from the Viewports coordinates to the screen coordinates of the containing window manager window.
```

Comment:
```suggestion
				Returns the transform from the Viewport's coordinates to the screen coordinates of the containing window manager window.
```

Sorry, another grammar issue I noticed 😅

Reply:
Thanks for the thorough review. Updated with the correction.